### PR TITLE
Event pictures

### PR DESCRIPTION
**Pull request by townsend3721**
Added photo uploading for events

### CODE REVIEW

File: spec/controllers/events_controller_spec.rb [hunk 1]
```diff
@@ -3,13 +3,14 @@
 describe EventsController do
   let!(:test_event) { FactoryGirl.create(:event, id: 1) }
   let!(:fut_event) { FactoryGirl.create(:future_event, id: 2) }
```
Comment (harperiscoding) [inline spec/controllers/events_controller_spec.rb]: try to only put let statements in blocks they're necessary in. check out my branch for some ideas.

Comment (harperiscoding) [reply spec/controllers/events_controller_spec.rb]: this will prevent you from having to change the expect statement for index as well.

**Review by harperiscoding — CHANGES_REQUESTED** [review]

**Comment by dantoth24** [comment]
image 
This error should not be displayed to the user. Also, the image I tried to upload was a 400x400 jpg. 

Edit: Got it to work by installing ImageMagic. Make sure to take note what external software is needed.

**Comment by dantoth24** [comment]
image
To make this UI more clear remove photo should be next to the checkmark instead of the image.

image
Remove the table row name Picture: from event pages

**Comment by townsend3721** [comment]
Fixed the clunkiness, added stuff to the readme for downloading image kick

**Pull request closed without merging**